extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {poll} from '../../../../../testing/iframe';
+
+describe.configure().run('amp-lightbox-gallery', function() {
+  this.timeout(5000);
+  const extensions = [
+    'amp-lightbox-gallery',
+    'amp-image-viewer',
```

Comment:
for a realistic sample, we should not include `amp-image-viewer` extension. Does it cause issues if not included?

Reply:
Yes, I think they are issue related to our build. The testing/build infrastructure appears to try to fetch a particular version from the CDN for extensions where we call `installExtensionForDoc`. I'm not quite sure why.

Reply:
Okay, I kind of figured it out. It's because our integration testing infrastructure's `env.win` doesn't have `localDev` set to true. I overrode it in my test so it stops trying to fetch from CDN. I wonder if this is a flag that should be set to true more generally in the `describes.integration` infrastructure. @rsimha would you have an opinion on the best way to do this?

Reply:
We already set `localDev` to true after `gulp build` here:

https://github.com/ampproject/amphtml/blob/7d307ab8a12e4d58360d430b2d61327e39b4846b/gulpfile.js#L1204-L1210

What this does is:

https://github.com/ampproject/amphtml/blob/7d307ab8a12e4d58360d430b2d61327e39b4846b/gulpfile.js#L797-L812

Are there files other than `amp.js` and `integration.js` that need `localDev` to be set to true?

Reply:
Yes. `<amp-lightbox-gallery>` is using `installExtensionForDoc`, which traces down to a call to 

```
const scriptSrc = calculateExtensionScriptUrl(loc, extensionId,
        opt_extensionVersion, getMode().localDev);
```
I'm noticing that my tests fail because they are trying to fetch extensions from `cdn.ampproject.org` instead of localhost because `getMode().localDev` is returning false, unless I explicitly set the `localDev` variable to true (as done in this PR). 

I'm running this with `gulp test --files=...`, and have tried doing a full gulp build ahead of time, but when I step through the code there, `localDev` is set to false.